python/mleap/sklearn/preprocessing/data.py

```diff
@@ -651,20 +667,19 @@ def serialize_to_bundle(self, transformer, path, model_name):
     def deserialize_from_bundle(self, transformer, node_path, node_name):
 
         attributes_map = {
-            'size': 'n_values_'
+            'size': 'categories_',
+            'handle_invalid': 'handle_unknown',
         }
 
         full_node_path = os.path.join(node_path, node_name)
         transformer = self.deserialize_single_input_output(transformer, full_node_path, attributes_map)
 
-        # Set Sparse = False
-        transformer.sparse = False
+        transformer.categories_ = np.asarray([range(transformer.categories_)])
+        if transformer.handle_unknown == 'keep':
+            transformer.handle_unknown = 'ignore'
+        transformer.drop_idx_ = None
 
-        # Set Feature Indices
-        n_values = np.hstack([[0], [transformer.n_values_]])
-        indices = np.cumsum(n_values)
-        transformer.feature_indices_ = indices
-        transformer.active_features_ = range(0, transformer.n_values_)
+        transformer.sparse = False
```

Comment:
This is potentially problematic because the default value for the `sparse` parameter in `OneHotEncoder.__init__` is `True`. So we would be silently changing this during deserialization. I don't know if this is better or worse than erroring every time users don't explicitly pass `sparse=False`, because that would be really annoying

Reply:
yeah, not sure about this. i think for now, maybe we could leave it as is setting `sparse=False` and see if it becomes an issue.